action.yml

```diff
@@ -14,6 +14,12 @@ inputs:
   environment:
     description: "Orchestra environment to run the pipeline with"
     required: false
+  task_ids:
+    description: "Orchestra task IDs to run"
+    required: false
+  continue_downstream_run:
+    description: "Continue downstream run boolean"
```

Comment:
`Continue running downstream tasks when specifying task_ids to run`

this makes it clearer it's used with the task_ids prop